src/DotLiquid/Template.cs

```diff
@@ -212,10 +212,10 @@ public static void RegisterValueTypeTransformer(Type type, Func<object, object>
         /// </summary>
         /// <param name="source"></param>
         /// <returns></returns>
-        public static Template Parse(string source)
+        public static Template Parse(string source, IRenderableFactory renderableFactory = null)

```

Comment:
This runs contrary to the standing pattern for this class. Configuration / customization is not done via additional parameters but why global registration. We should remains consistent with the existing pattern unless we have specific need not to.